Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -821,12 +821,16 @@ def get_offense_types(client: Client, offenses: List[Dict]) -> Dict:
     Returns:
         (Dict): Dictionary of {offense_type_id: offense_type_name}
     """
-    offense_types_ids = {offense.get('offense_type') for offense in offenses if offense.get('offense_type') is not None}
-    if not offense_types_ids:
-        return dict()
-    offense_types = client.offense_types(filter_=f'''id in ({','.join(map(str, offense_types_ids))})''',
-                                         fields='id,name')
-    return {offense_type.get('id'): offense_type.get('name') for offense_type in offense_types}
+    try:
+        offense_types_ids = {offense.get('offense_type') for offense in offenses if offense.get('offense_type') is not None}
+        if not offense_types_ids:
+            return dict()
+        offense_types = client.offense_types(filter_=f'''id in ({','.join(map(str, offense_types_ids))})''',
+                                             fields='id,name')
+        return {offense_type.get('id'): offense_type.get('name') for offense_type in offense_types}
+    except Exception as e:
+        demisto.info(f"Encountered an issue while getting offense type: {e}")
```

Comment:
@DeanArbel why not demisto.error?

Reply:
> @DeanArbel why not demisto.error?

We wanted something more than `debug`.
`error` is indeed more suitable here.